### DISCUSSION TITLE
id for Forum Header

### DISCUSSION THREAD

**Discussion by nicksaahs** [POPWorldMedia/POPForums#256] [Ideas]
Would you be so kind to give the forum header an id (like the footer)?
When embedding the forums, it doesn't make sense to have double headers and footers, so it makes sense to hide them, and right now it's just a kludge to identify it. You've made it easy enough to copy the necessary bits to my own header/footer.
I could probably do this in a custom build, but I'm convinced this is added value for all users.

**Comment by jeffputz** [comment]
What do you mean "header?" You mean the navigation?

**Reply by nicksaahs** [reply]
Yes, indeed.
In `PopForumsMaster.cshtml`, I find: `<nav class="navbar navbar-expand-lg navbar-light bg-light">`.
It would be great if the nav had an id (likely: ForumHeader of ForumNav), like the footer: `<div id="ForumFooter" class="col-12 mt-3">`.

**Comment by jeffputz** [comment]
I mean, it's the only `nav` in `#ForumContainer`, so it's a straight forward selector.

**Reply by nicksaahs** [reply]
You are right indeed. I've learned something new - as simple as it may seem to you.
Either way: you could still consider adding it, as it would make the selector more likely to be stable. By all means feel free not to.
Thanks again for your input
